### PR TITLE
Adding more bus routes!

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

Adding more and more bus routes to the dashboard

## Changes

- Adds a large collection of new bus routes to the dashboard

### CODE REVIEW

File: server/bus/gen_bus_data.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 routes="$@"
 if [ -z "$routes" ]; then
```
Comment (rudiejd) [inline server/bus/gen_bus_data.sh:4]: i feel like we could be storing these in a csv at this point 😿

**Review by rudiejd — APPROVED** [review]

File: server/chalicelib/date_utils.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 EASTERN_TIME = ZoneInfo("US/Eastern")
 
 # The most recent date for which we have monthly data
-MAX_MONTH_DATA_DATE = "2024-03-31"
+MAX_MONTH_DATA_DATE = "2024-04-30"
```
Comment (rudiejd) [inline server/chalicelib/date_utils.py:11]: this will have to change

File: server/bus/gen_bus_data.sh (hunk 1 above)
Comment (devinmatte) [reply server/bus/gen_bus_data.sh:4]: This whole process needs to be improved yeah.

I also need to move this whole process to mbta-performance repo

File: server/chalicelib/date_utils.py (hunk 2 above)
Comment (devinmatte) [reply server/chalicelib/date_utils.py:11]: Once we have monthly data up to June, we can update the PR and merge

Comment (devinmatte) [reply server/chalicelib/date_utils.py:11]: Need to merge https://github.com/transitmatters/t-performance-dash/pull/992 first

**Review by nathan-weinberg — APPROVED** [review]

**Pull request merged**